### DISCUSSION TITLE
licensing / EULA topic

### DISCUSSION THREAD

**Discussion by Bachzn** [AdguardTeam/AdGuardHome#2897] [Q&A]
HI Team,

I had a email conversation with the AdGuard Team yesterday around the use of AdGuard Home in a school and they advised me to put my request into Github.

It's more or less a licensing question. As far as I can see AdGuard Home is an open source product. True? Does an EULA exist? Are there plans to make AdGuard Home commercial in the future? I have to clarify this before I can make the next step to implement this in a small school to secure the access for the kids.  

What would you recommend is such a case?

best regards
Chris

**Comment by ameshkov** [comment]
Hello Chris,

AdGuard Home is indeed an open-source product. There's no separate EULA for it and we don't make you agree with anything, you can use it as is as per GPL license.

Regarding making it commercial, we have no such plans. In the future, we may add better integration with existing AdGuard products, but that would be optional anyway, the core functionality stays as it is.
